File: in_java/Pattern_16.java
package in_java;

public class Pattern_16 extends Pattern {
    public Pattern_16(int row, int column, String symbol) {
        super(row, column, symbol);
    }

    public void print() {
        int index = 0;
        for (int i = 0; i < this.row; i++) {
            for (int j = 0; j < this.column; j++) {
                if (j == index) {
                    System.out.print("\\");
                } else if (j == this.column - 1 - index) {
                    System.out.print("/");
                } else {
                    System.out.print(this.symbol);
                }
            }
            index++;
            System.out.println();
        }
    }
}

File: in_go/src/pattern_16.go
package src

import "fmt"

func Pattern_16() {
	row := 7
	column := 7
	symbol := "*"
	index := 0
	for i := 0; i < row; i++ {
		for j := 0; j < column; j++ {
			if j == index {
				fmt.Print("\\")
			} else if j == column-1-index {
				fmt.Print("/")
			} else {
				fmt.Print(symbol)
			}
		}
		fmt.Println()
		index++
	}
}
